### PR TITLE
hw 1.3

### PR DESCRIPTION
**Pull request by Djeckkenni**

### CODE REVIEW

File: src/main/java/core/basesyntax/BinaryString.java [hunk 1]
```diff
@@ -7,6 +7,18 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder binaryString = new StringBuilder("");
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/BinaryString.java]:
```suggestion
        StringBuilder binaryString = new StringBuilder();
```

File: src/main/java/core/basesyntax/BinaryString.java [hunk 2]
```diff
@@ -7,6 +7,18 @@
...
+        while (value > 0) {
+            if ((value % 2) == 0) {
+                binaryString.append(0);
+            } else {
+                binaryString.append(1);
+            }
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/BinaryString.java]: binaryString.append(value % 2);
```suggestion
            binaryString.append(value % 2);
```

File: src/main/java/core/basesyntax/BinaryString.java (hunk 1 above)
Comment (Djeckkenni) [reply src/main/java/core/basesyntax/BinaryString.java]: Done

File: src/main/java/core/basesyntax/BinaryString.java (hunk 2 above)
Comment (Djeckkenni) [reply src/main/java/core/basesyntax/BinaryString.java]: Done

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**